From: Gunnar Hjalmarsson <gunnarhj@ubuntu.com>
Date: Tue, 7 Aug 2018 11:15:57 +0100
Subject: Don't set language environment

Setting of the users' language environment is handled by PAM in
Ubuntu. The settings are stored in ~/.pam_environment, and PAM's
reading of that file is triggered by /etc/pam.d/gdm-password or
/etc/pam.d/gdm-autologin.
Bug-Ubuntu: https://launchpad.net/bugs/1662031
Forwarded: not-needed
---
 daemon/gdm-session.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/daemon/gdm-session.c b/daemon/gdm-session.c
index d686e35..71095ba 100644
--- a/daemon/gdm-session.c
+++ b/daemon/gdm-session.c
@@ -2698,7 +2698,7 @@ set_up_session_environment (GdmSession *self)
                 gdm_session_set_environment_variable (self, "XDG_CURRENT_DESKTOP", desktop_names);
         }
 
-        set_up_session_language (self);
+        // set_up_session_language (self);
 
         display_mode = gdm_session_get_display_mode (self);
         if (display_mode == GDM_SESSION_DISPLAY_MODE_REUSE_VT) {
